pycode/memilio-epidata/memilio/epidata/getCommuterMobility.py

```diff
@@ -211,6 +209,12 @@ def get_commuter_data(read_data=dd.defaultDict['read_data'],
         commuter_migration_file = gd.get_file(
             '', url, False, param_dict, interactive=True)
 
+        # merge eisenach and wartburgkreis
```

Comment:
merge eisenach has to be True here because our population data file does not provide data for eisenach. In this case it would result in a ZeroDivisionError in line 281